src/components/SiteNav/styles.css

```diff
@@ -4,18 +4,68 @@
 	font-weight: bold;
 }
 
-#site-nav ul {
+#site-nav a { color: white; }
+
+#logo-rounded {
+	border-radius: var(--size-2);
+}
+
+#nav-menubar {
 	gap: var(--size-2);
 	align-items: center;
 	padding: var(--size-3);
 }
 
-#site-nav a { color: white; }
+#nav-menubar .mobile-only {
+	display: none;
+}
 
-#logo-rounded {
-	border-radius: var(--size-2);
+@media (max-width: 500px) {
```

Comment:
Let's use `rem` for media queries as it provides better support for zooming text. Reference: https://www.youtube.com/watch?v=xCSw6bPXZks